services-core/src/gas_price/ethgasstation.rs

```diff
@@ -0,0 +1,79 @@
+use super::{linear_interpolation, GasPriceEstimating};
+use crate::http::{HttpClient, HttpFactory, HttpLabel};
+use anyhow::Result;
+use ethcontract::U256;
+use isahc::http::uri::Uri;
+use pricegraph::num;
+use std::{convert::TryInto, time::Duration};
+
+// Gas price estimation with https://ethgasstation.info/ , api https://docs.ethgasstation.info/gas-price .
```

Comment:
Do we need, like the Gnosis Safe gas station, have different URLs for mainnet/rinkeby? At least something along the lines of:
```rust
pub fn api_url_from_network_id(network_id: &str) -> Option<&'static str> {
    match network_id {
        "1" => Some(API_URI),
        _ => None,
    }
}
```

Reply:
Yes, this and the GasNow gas station don't support rinkeby. When I integrate them so that they are actually used I'll add a check that the network id is mainnet.